app/views/trainees/training_details/edit.html.erb

```diff
@@ -12,9 +12,20 @@
 
       <h1 class="govuk-heading-l"><%= t("views.forms.training_details.title") %></h1>
 
-      <%= f.govuk_date_field :commencement_date, legend: {
-        text: t("views.forms.training_details.commencement_date.label"), size: "s"
-      }, hint: { text: t("views.forms.training_details.commencement_date.hint_html", year: Time.zone.now.year) } %>
+      <% if @training_details_form.course_start_date %>
+        <%= f.govuk_radio_buttons_fieldset(:commencement_date_option, legend: { text: t("views.forms.training_details.commencement_date_option.label"), size: "s" }, hint: { text: t("views.forms.training_details.commencement_date_option.hint", year: Time.zone.now.year) }) do %>
+          <%= f.govuk_radio_button :commencement_date_option, TrainingDetailsForm::COMMENCEMENT_DATE_OPTION_COURSE, checked: (@training_details_form.commencement_date_option == TrainingDetailsForm::COMMENCEMENT_DATE_OPTION_COURSE), value: true, label: { text: t("views.forms.training_details.commencement_date_option.course", date: @training_details_form.course_start_date.strftime("%e %B %Y")) } %>
+          <%= f.govuk_radio_button :commencement_date_option, TrainingDetailsForm::COMMENCEMENT_DATE_OPTION_MANUAL, checked: (@training_details_form.commencement_date_option == TrainingDetailsForm::COMMENCEMENT_DATE_OPTION_MANUAL), label: { text: t("views.forms.training_details.commencement_date_option.manual") } do %>
+            <%= f.govuk_date_field :commencement_date, legend: {
+              text: t("views.forms.training_details.commencement_date.label2"), size: "s"
+            }, hint: { text: t("views.forms.training_details.commencement_date.hint2", year: Time.zone.now.year) } %>
```

Comment:
I wonder if this translation name could be more descriptive vs having `2` appended.